README.md

```diff
@@ -35,7 +40,12 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+var Instructor = mongoose.model('Instructor', schema);
+
+Instructor.findOne({"name": Andy}, function (err, andy) {
+  console.log(andy);
+  andy.wishlist_items.create({"description": "Resin Laying Deer Figurine, Gold")
```

Comment:
Close! Try `.push` instead. `.create` is only on the class itself (e.g. `Instructor.create`).